data/scripts/movements/closing_door.lua

```diff
@@ -22,7 +22,7 @@ function closingDoor.onStepOut(creature, item, position, fromPosition)
 	while tileItem and i < tileCount do
 		tileItem = tile:getThing(i)
 		if tileItem and tileItem:getUniqueId() ~= item.uid and tileItem:getType():isMovable() then
```

Comment:
can't use position or fromPosition from function parameters, they returns always the same positions, creature position is good becouse it's should be always valid